test/e2e/specs/editor/various/style-variation.spec.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * WordPress dependencies
+ */
+const { test, expect } = require( '@wordpress/e2e-test-utils-playwright' );
+
+test.describe( 'adding blocks', () => {
+	test( 'Should switch to the plain style of the quote block', async ( {
+		page,
+		pageUtils,
+	} ) => {
+		await pageUtils.createNewPost();
+
+		// Inserting a quote block
+		await pageUtils.insertBlock( { name: 'core/quote' } );
+		await page.keyboard.type( 'Quote content' );
```

Comment:
Nitpick: maybe we can inline the content directly inside the block? (Didn't test it)

```suggestion
		await pageUtils.insertBlock( { name: 'core/quote', attributes: { content: 'Quote content' } } );
```

Reply:
```
await pageUtils.insertBlock( {
  name: 'core/quote',
  attributes: { content: 'Quote content' },
} );
```
does not seem to work for the quote block.

Reply:
Looks like it should be:

```js
await pageUtils.insertBlock( { name: 'core/quote', attributes: { value: '<p>Quote content</p>' }, } );
```